packages/danger/src/PullRequestManager.ts

```diff
@@ -64,7 +64,7 @@ export class PullRequestManager {
     const changelogEntries = {
       [DEFAULT_CHANGELOG_ENTRY_KEY]: {
         type: ChangelogEntryType.BUG_FIXES,
-        message: this.prTitle.replace(/^(\[[^\]]\])+/g, '').trim(),
+        message: this.prTitle.replace(/\[.*\]/, '').trim(),
```

Comment:
Hmm, I guess we don't want to strip everything till the last `]` 🤔 I think previous regexp was fine but it's missing `*`.

```suggestion
        message: this.prTitle.replace(/\[[^\]]*\]/g, '').trim(),
```